@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -198,6 +190,25 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
             }
             virtualFocus.moveFocusUp();
           }
+        } else if (e.key === "Home") {
+          e.preventDefault();
+          toggleIsListOpen(false);
+          virtualFocus.moveFocusToTop();
+        } else if (e.key === "End") {
+          e.preventDefault();
+          toggleIsListOpen(true);
+          virtualFocus.moveFocusToBottom();
+        } else if (e.key === "PageUp") {
+          e.preventDefault();
+          virtualFocus.moveFocusUpBy(6);
+          return false;
+        } else if (e.key === "PageDown") {
+          e.preventDefault();
+          if (virtualFocus.activeElement === null || !isListOpen) {
```

Comment:
Do we need to check this for `} else if (e.key === "End")` to? `toggleIsListOpen(true);` is always called now when `end` is pressed

Reply:
Hmm. Maybe. I am unsure if there is a problem with actually doing this. One thing is ArrowDown/-Up, which the user could press-and-hold, but Home/End and PageUp/-Down shouldn't cause a performance hit even IF this causes any updates.

Another thing is an inconsistency where ArrowUp from the first element and pressing Home will both close the list, whereas PageUp will stop on the first element. Should we make this more consistent?